Encounter Me/wwwroot/css/app.css

```diff
@@ -249,6 +249,21 @@ mapText {
 }
 
 
+@property --num {
+    syntax: "<integer>";
+    initial-value: 0;
+    inherits: false;
+}
+
+.counter {
+    transition: --num 20s;
+    counter-reset: num var(--num);
+}
+.counter::after {
+    content: counter(num);
+}
+
+
```

Comment:
Would be nice, if you would somehow comment on your own created CSS classes, because currently I am working on UI and will have to modify app.css file.

Reply:
Will try to do that in future commits